content/cpp/concepts/math-functions/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'sqrt()'
+Description: 'Returns square root of the argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`sqrt()`** function returns the square root of the argument.
+
+## Syntax
+
+```cpp
+sqrt(x)
+```
+
+If the parameter `x` is negative (less than 0), then a domain error will occur.
+
+## Example
+
+The following example uses the `sqrt()` function to find the square root of `4`:
+
+```cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+  double x = 4;
+  double result;
+
+  result = std::sqrt(x);
+
+  std::cout << "The square root of " << x << " is " << result << "\n";
+  // Output: The square root of 4 is 2
+}
+```
+
+## Codebyte Example
+
+The following example is runnable and returns the square root of `9` with the `sqrt()` function:
```

Comment:
```suggestion
The following example is runnable and returns the square root of `9` with the `sqrt()` function:

```

Reply:
It's saying the suggestion is identical

Reply:
Ah, that it is!

Reply:
Actually, this suggestion still stands. In a [previous suggestion](https://github.com/Codecademy/docs/pull/974/files#r966240255), it recommend empty above and below line # 45. However, there is still no empty line between # 45 and # 46.

Reply:
If you still run into issues, I'm happy to move in and make this change 😄

Reply:
It won't let me just commit your suggestion, saying it's identical

Reply:
No worries! Made the change in 350a39158608949bb323f0007f280a6de6359e76. All the checks should pass now 😄